docs/topics/forms/formsets.txt

```diff
@@ -287,12 +287,20 @@ sure you understand what they do before doing so.
 a form instance with a prefix of ``__prefix__`` for easier use in dynamic
 forms with JavaScript.
 
+.. _formsets-error-messages:
+
 ``error_messages``
 ------------------
 
 The ``error_messages`` argument lets you override the default messages that the
 formset will raise. Pass in a dictionary with keys matching the error messages
-you want to override. For example, here is the default error message when the
+you want to override. Error message keys include ``'too_few_forms'``,
+``'too_many_forms'``, and ``'missing_management_form'``. To format
+``'too_few_forms'`` and ``'too_many_forms'`` error messages they must
```

Comment:
Hey @hop ! Thanks for the original patch and your comment. I included your suggestion here! :+1: